addon/components/polaris-text-style.js

```diff
@@ -40,12 +63,24 @@ export default Component.extend({
    */
   text: null,
 
+  dataTestTextStyle: true,
```

Comment:
passing this for tagless component cause passing something like `data-test-*` will result in a warning